### DISCUSSION TITLE
Deleting all files from mount leaves files on some volumes

### DISCUSSION THREAD

**Discussion by apunkt** [seaweedfs/seaweedfs#5776] [Q&A] [answered]
Hi,

am new to SeaweedFS, evaluating GlusterFS replacement.

My simple test screnario:
1 Master:
`./weed master -volumeSizeLimitMB=1024`
1 Volume:
`./weed volume -max=0 -mserver=192.168.0.206:9333 -dir=./brick -ip=192.168.0.206 -port=8090`
1 Filer:
`./weed volume -max=0 -mserver=192.168.0.206:9333 -dir=./brick -ip=192.168.0.206 -port=8090`
1 Mount:
`./weed mount -filer=192.168.0.206:8888 -dir=/mnt/tmp`

to check performance on different sized files I use dd on the mount:
`dd if=/dev/random of=/mnt/tmp/dafile3.img bs=1M count=4096`

file gets chunked and spread across volumes.

Now doing:
`rm /mnt/tmp/*`

![grafik](https://github.com/user-attachments/assets/f427b8ae-f30b-4e06-934c-d989809c92cc)

which is (almost) as expected. See volumes 3,5,6,7

Doing
`volume.vacuum -garbageThreshold=0`

Expectation is: _clean volumes!_
**but**
![grafik](https://github.com/user-attachments/assets/7b188d33-eeaa-46f6-a481-22538df5e330)

cannot get rid of these anymore.
Why?

Doing:
```
> volume.fsck
total 3 directories, 0 files
dataNode:192.168.0.206:8090	volume:5	entries:1	orphan:1	100.00%	62624B
dataNode:192.168.0.206:8090	volume:3	entries:1	orphan:1	100.00%	117620B
dataNode:192.168.0.206:8090	volume:6	entries:1	orphan:1	100.00%	31268B
dataNode:192.168.0.206:8090	volume:7	entries:1	orphan:1	100.00%	125093B

Total		entries:4	orphan:4	100.00%	336605B
This could be normal if multiple filers or no filers are used.

```

Does this mean it is ok?
Will this sum up over time when doing fops?
Also, using exactly one filer!?
Why volume 2 is _not_ mentioned after all?

**Comment by chrislusf** [comment]
`garbageThreshold = 0` means using the default value.

**Comment by apunkt** [comment, marked as answer]
Thanks for letting me know!

However, the expectation would be that these files have been deleted in the first vacuum then, too.

Learned about orphans:
What worked was a 
`volume.fsck -reallyDeleteFromVolume`
